### PR TITLE
Add information related to DLT_viewer and hints

### PR DESCRIPTION
**Pull request by pancio88**

### CODE REVIEW

File: docs/articles/sde/introduction_to_sde.rst
Comment (tobsan) [inline docs/articles/sde/introduction_to_sde.rst]: If one has to add extra repositories for the SDE to work, please describe briefly the steps to get it working.

File: docs/articles/sde/introduction_to_sde.rst
Comment (tobsan) [inline docs/articles/sde/introduction_to_sde.rst]: I believe the tools are called Qt Creator and DLT Viewer.

**Review by tobsan — CHANGES_REQUESTED** [review]
Please add "dlt" to the spelling_wordlist as well.

File: docs/articles/sde/introduction_to_sde.rst
Comment (pancio88) [reply docs/articles/sde/introduction_to_sde.rst]: You think is a good idea to flood with apt-get commands or links to the proper repositories this page? If so I will updated but, for my understanding, the page should easy to read, my note is just a warning that the previous procedure is not straight forward for Ubuntu users.

File: docs/articles/sde/introduction_to_sde.rst
Comment (pancio88) [reply docs/articles/sde/introduction_to_sde.rst]: You right, QtCreator is also listed on the spelling_wordlist.

File: docs/articles/sde/introduction_to_sde.rst
Comment (tobsan) [reply docs/articles/sde/introduction_to_sde.rst]: Perhaps you can just mention what repos one has to add to get the latest version?

Comment (pancio88) [reply docs/articles/sde/introduction_to_sde.rst]: I went to the websites and download the latest debian package.

Comment (tobsan) [reply docs/articles/sde/introduction_to_sde.rst]: Okay. I don't think I understand exactly the problem.

* Does it work in Ubuntu even with older versions of the components, or are newer versions required?
* If newer versions are required, can you update the PR with how to get them?

Comment (pancio88) [reply docs/articles/sde/introduction_to_sde.rst]: Sorry if I don't explain well.
* if you use the default apt-get install vagrant/virtualbox you get older version that are not compatible with the git vagrant repository. You will get errors and freeze your PC sometimes. I don't know from which version will work properly, I just know that with the latest version available they works fine.
* To get the latest version I went to vagrant/virtualbox website and get the debian package to install them (instead of use the apt-get from a remote repos). Do you mean that I should add the links of the official vagrant/virtualbox distro?

Comment (tobsan) [reply docs/articles/sde/introduction_to_sde.rst]: Yes, a link to the VBox and Vagrant sites, I guess linking to these two would suffice:

* https://www.virtualbox.org/wiki/Linux_Downloads (even mentions how to add to apt)
* https://www.vagrantup.com/downloads.html (only has raw .deb files it seems)

**Comment by pancio88** [comment]
@tobsan, I've done your request. Can I ask you how I can add othere reviewers? FYI @jonte @breakreturn

**Review by tobsan — APPROVED** [review]

**Comment by tobsan** [comment]
@pancio88 I've added @jonte and @breakreturn

**Review by breakreturn — APPROVED** [review]

**Comment by breakreturn** [comment]
Approved by me as well, but I guess our guidelines say there should be a signed-off-by line in the commit message. Maybe I'm misreading it but I can't see it.

**Comment by tobsan** [comment]
@breakreturn no that is correct. @pancio88 please add a signed-off-by line (git commit -s)

**Comment by pancio88** [comment]
Done

**Comment by GordanM** [comment]
@pancio88 and @tobsan I believe that the DLT viewer is not included in the mainline SDE but the project specific one and as such this might not be appropriate for the blueprint. What do you think?

Edit: Ideally the mainline SDE should have DLT viewer included and then the change would be perfectly fine :).

**Comment by breakreturn** [comment]
Oh. In that case this change is not appropriate no. But the inclusion of DLT Viewer would be welcome upstream though.

**Comment by GordanM** [comment]
@breakreturn agreed. Blocking this change set until the mainline SDE has DLT Viewer installed.

**Review by GordanM — CHANGES_REQUESTED** [review]
Mainline SDE doesn't have DLT Viewer installed by default.

**Review by breakreturn — CHANGES_REQUESTED** [review]
Is DLT Viewer part of the SDE?

**Comment by pancio88** [comment]
I merged the pull request including the DLT viewer 2 days ago. Is part of the SDE  (http://ntg-x.projects.luxoft.com/gitlab/ntg-x/sde/merge_requests/3)

**Comment by pancio88** [comment]
@GordanM  why you delete my comment?

**Comment by pancio88** [comment]
@breakreturn @GordanM  yes, DLT_Viewer is part of the SDE since few days ago.

**Comment by rpannek** [comment]
@pancio88 The PELUX SDE is maintained here https://github.com/pelagicore/pelux-sde I know that you're working on a downsteam project which uses a fork of that. This blueprint is meant to be used by many different downstream projects.

**Comment by rpannek** [comment]
Sorry, pressed the wrong button while commenting.

**Comment by pancio88** [comment]
@rpannek  I don't get in this case which should be the next steps. I should update the Pelagicore/pelux-sde or I should close this merge request and open a new one related just to out project?

**Comment by GordanM** [comment]
@pancio88 both of those are valid solutions. It depends what you are more keen on doing. I would prefer you to update the mainline SDE though.

**Comment by pancio88** [comment]
@GordanM my only concern if I should update the mainline SDE of if is something that integration team should do, since I'm not sure if the only updates are mine or from someone else.

**Comment by GordanM** [comment]
@pancio88 there is no concept of an integration team in the open source PELUX context. The reviewers (@tobsan , me , etc.) will approve your request (this one) if the the patch that adds the installation of  DLT Viewer is merged to the pelux-sde.

**Comment by pancio88** [comment]
PR https://github.com/Pelagicore/pelux-sde/pull/8  with DLT viewer is merged in Pelagicore/SDE. @GordanM, @breakreturn, can this PR be approved/merged now?

**Comment by pancio88** [comment]
@GordanM , @breakreturn  check it now, please.

**Review by GordanM — APPROVED** [review]

**Review by breakreturn — APPROVED** [review]

**Pull request merged**
